src/trellis/platforms/browser/__init__.py

```
@@ -12,6 +12,8 @@
     # TrellisApp registers the bridge, then executes user code
 """
 
+# Register trellis-browser module with the bundler
+from trellis.platforms.browser import _register as _  # noqa: F401
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)